curriculum/fullstack-app/schedule.md

```diff
@@ -1,33 +1,40 @@
-# Single-page app Week Schedule
+# Single Page App Week Schedule
```

Comment:
@jackherizsmith why remove the hyphen 🤷 ?

Reply:
No idea I was just getting confused by all the hyphens. It's called fullstack app in the folder and url, is there a reason we use both in different places?

Reply:
It should probs be changed to single-page app everywhere tbh. Fullstack app is a weird term that doesn't really exist